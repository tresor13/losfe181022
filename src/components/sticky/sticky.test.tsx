import renderer from 'react-test-renderer';
import Sticky from './index';

it('Sticky should render correct', () => {
  const component = renderer.create(<Sticky image="test" position="top"/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


it('Props should be correct', () => {
  const component = renderer.create(<Sticky position="top" image="test"/>);
  expect(component.root.props?.image).toEqual('test');
  expect(component.root.props?.position).toEqual('top');
});
