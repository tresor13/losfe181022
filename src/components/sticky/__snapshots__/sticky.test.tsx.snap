// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sticky should render correct 1`] = `
<div
  className="sticky"
  style={
    Object {
      "backgroundImage": "url(test)",
      "top": 0,
    }
  }
/>
`;
